harmonic/ros2_overview.md

```diff
@@ -57,12 +57,17 @@ ROS nodes will be intraprocess.
 
 This figure illustrates the concept of composition. The left diagram captures
 the idea of not using composition. All the three example nodes are standalone
-nodes, and they can talk via interprocess communication. The center diagram
-represents the scenario where we use composition and we start our own container
-from our own launch file. All communication is intraprocess here. The right
-diagram is still using composition but the launch file doesn't start the
-container directly. This setup by itself will not work until you start an
-external ROS container (manually or via a separate launch file).
+nodes, and they can talk via interprocess communication using the bridge.
+The center diagram represents the scenario where we can use composition and our
+own ROS container between Gazebo and the bridge, but there's an additional
```

Comment:
Reading this, it's not clear which perspective "own" refers to. That is, as a reader, I might think my own ROS container, but this is referring to `ros_gz`'s container.

```suggestion
The center diagram represents the scenario where we can use composition with a ROS container created by a `ros_gz` launch file containing both Gazebo and the bridge, and an additional
```